ert_storage/endpoints/experiments.py

```diff
@@ -93,3 +93,10 @@ async def get_experiment_metadata(
     """
     experiment = db.query(ds.Experiment).get(experiment_id)
     return experiment.metadata_dict
+
+
+@router.post("/experiments/{experiment_id}/delete")
```

Comment:
```suggestion
@router.delete("/experiments/{experiment_id}")
```

You then use HTTP's `DELETE` verb.

Reply:
of course 🙈